### DISCUSSION TITLE
Support cargo-binstall

### DISCUSSION THREAD

**Discussion by auronandace** [zellij-org/zellij#1067] [Ideas]
Zellij keeps on getting better and better. Unfortunately, this also tends to mean it gets bigger as more features are added. Sometimes those features bring in new dependencies and that increases the compile time of the project. In addition to being able to install zellij via `cargo install zellij` it might be advantageous to also support `cargo binstall zellij`.

Here is a link to cargo-binstall: https://github.com/ryankurte/cargo-binstall

As always I'd like to thank you for such an awesome project!

**Comment by jaeheonji** [comment]
Hi, @auronandace!

I like that! I also prefer to install only binary files in most cases.
We can install it through the package manager, but this is not official.

Using this we will be able to effectively serve only binary files officially.

**Reply by auronandace** [reply]
Well done with #1129. Thanks muchly!

**Reply by NewtonChutney** [reply]
I think this wasn't added into the docs..

**Reply by NewtonChutney** [reply]
Raised! https://github.com/zellij-org/zellij-org.github.io/pull/271
